src/pages/Header/Header.js

```diff
@@ -48,6 +73,14 @@ const Header = () => {
         <NavLink to="/feedback" activeClassName="active">
           FEEDBACK
         </NavLink>
+        <span className="line" />
+        {isAuthenticated ? (
+          <LogoutButton />
+        ) : (
+          <a href="/" onClick={handleLogin}>
```

Comment:
Shouldnt this be the login button component? if its just a handler button it should be a <Button> tag.